arcgis_map_sdk/lib/src/arcgis_map_controller.dart

```diff
@@ -149,6 +172,22 @@ class ArcgisMapController {
     );
   }
 
+  VoidCallback addStatusChangeListener(MapStatusListener listener) {
+    _listeners.add(listener);
+    return () => _listeners.removeWhere((l) => l == listener);
+  }
+
+  Future<void> reload() {
+    return ArcgisMapPlatform.instance.reload(mapId);
+  }
```

Comment:
Document methods

Reply:
I see that on the nativ side we are calling retryLoadAsync and I think it makes sense to maybe even call that method something like `retryLoad`.

The documentaton of `retryLoad` also explains that this will only load again if an error happend. 
We can just link to the arcigs docs and say a few simple words what it does👍🏽